### ISSUE TITLE
[google-cloud-compute][google-cloud-filestore] VpnGatewaysClient.create() and CloudFilestoreManagerClient.create() throw exceptions when updating the bom to 26.17.0

### ISSUE THREAD

**Issue by rnaval** [googleapis/google-cloud-java#9558]
#### Environment details

1. Affected APIs: google-cloud-compute (VpnGatewaysClient) and google-cloud-filestore (CloudFilestoreManagerClient) - possibly more
2. OS type and version: 
```
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
```
4. Java version: Java 11
5. Version(s):
```
        <groupId>com.google.cloud</groupId>
        <artifactId>libraries-bom</artifactId>
        <version>26.17.0</version>
```

#### Steps to reproduce

  1. Build using the 26.17.0 bom
  2. Create a client using either `VpnGatewaysClient` and `CloudFilestoreManagerClient`

#### Code example

```java
  public void start() {
    try {
      client = VpnGatewaysClient.create();
      request = AggregatedListVpnGatewaysRequest.newBuilder().setProject(getProjectId()).build();
...
    } catch (Exception e) {
      logger.get().error("{} Error when creating client instance {} : {}",
          this.getClass().getSimpleName(), projectName, e.getMessage());
    }
```

#### Stack trace
```
java.lang.IllegalAccessError: class com.google.cloud.compute.v1.VpnGatewayAggregatedList tried to access method 'com.google.protobuf.LazyStringArrayList com.google.protobuf.LazyStringArrayList.emptyList()' (com.google.cloud.compute.v1.VpnGatewayAggregatedList is in unnamed module of loader java.net.URLClassLoader @18078bef; com.google.protobuf.LazyStringArrayList is in unnamed module of loader 'app')
	at com.google.cloud.compute.v1.VpnGatewayAggregatedList.<init>(VpnGatewayAggregatedList.java:466)
	at com.google.cloud.compute.v1.VpnGatewayAggregatedList.<clinit>(VpnGatewayAggregatedList.java:2186)
	at com.google.cloud.compute.v1.stub.HttpJsonVpnGatewaysStub.<clinit>(HttpJsonVpnGatewaysStub.java:126)
	at com.google.cloud.compute.v1.stub.VpnGatewaysStubSettings.createStub(VpnGatewaysStubSettings.java:334)
	at com.google.cloud.compute.v1.VpnGatewaysClient.<init>(VpnGatewaysClient.java:149)
	at com.google.cloud.compute.v1.VpnGatewaysClient.create(VpnGatewaysClient.java:131)
	at com.google.cloud.compute.v1.VpnGatewaysClient.create(VpnGatewaysClient.java:123)
	at com.itrsgroup.collection.plugins.gcp.common.resourcemonitoring.vpn_gateway.GoogleCloudVpnGatewayMonitoringService.start(GoogleCloudVpnGatewayMonitoringService.java:43)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
	at com.itrsgroup.collection.plugins.gcp.common.service.GoogleCloudMonitoringServiceManager.start(GoogleCloudMonitoringServiceManager.java:192)
	at com.itrsgroup.collection.plugins.gcp.AbstractCommonCollector.doStart(AbstractCommonCollector.java:29)
	at com.itrsgroup.platform.daemon.core.AbstractComponent.start(AbstractComponent.java:186)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
	at com.itrsgroup.collection.ca.CollectionAgent.doStart(CollectionAgent.java:112)
	at com.itrsgroup.platform.daemon.core.AbstractComponent.start(AbstractComponent.java:186)
	at com.itrsgroup.platform.daemon.config.ConfigLoader.loadAndStart(ConfigLoader.java:117)
	at com.itrsgroup.collection.ca.Main.loadAndStart(Main.java:48)
	at com.itrsgroup.collection.ca.Main.lambda$main$0(Main.java:37)
	at com.itrsgroup.platform.daemon.DaemonMain.daemonMain(DaemonMain.java:105)
	at com.itrsgroup.platform.daemon.DaemonMain.daemonMain(DaemonMain.java:67)
	at com.itrsgroup.collection.ca.Main.main(Main.java:33)
```

#### Any additional information below

Rolling back to 26.16.0 fixes the issue.

Specifically, rolling back the compute and filestore libraries to:
```
com.google.api.grpc:proto-google-cloud-compute-v1:1.28.0
com.google.api.grpc:proto-google-cloud-filestore-v1:1.19.0
```

Seems to be breaking when using
```
com.google.api.grpc:proto-google-cloud-compute-v1:1.29.0
com.google.api.grpc:proto-google-cloud-filestore-v1:1.20.0
```

**Comment by suztomo** [comment]
Trying to reproduce the problem...

**Comment by suztomo** [comment]
Main.java:

```
package org.example;

import com.google.cloud.compute.v1.AggregatedListVpnGatewaysRequest;
import com.google.cloud.compute.v1.VpnGatewaysClient;

public class Main {

  public static void main(String[] args) throws Exception {
    VpnGatewaysClient client = VpnGatewaysClient.create();
    AggregatedListVpnGatewaysRequest request = AggregatedListVpnGatewaysRequest.newBuilder()
        .setProject("12345").build();
    System.out.println("client: " + client + ", request: " + request);
  }
}
```

```
<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0"
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
  <modelVersion>4.0.0</modelVersion>

  <groupId>org.example</groupId>
  <artifactId>i9558</artifactId>
  <version>1.0-SNAPSHOT</version>

  <dependencyManagement>
    <dependencies>
      <dependency>
        <groupId>com.google.cloud</groupId>
        <artifactId>libraries-bom</artifactId>
        <version>26.17.0</version>
        <scope>import</scope>
        <type>pom</type>
      </dependency>
    </dependencies>
  </dependencyManagement>

  <dependencies>
    <dependency>
      <groupId>com.google.cloud</groupId>
      <artifactId>google-cloud-compute</artifactId>
    </dependency>
  </dependencies>

  <properties>
    <maven.compiler.source>17</maven.compiler.source>
    <maven.compiler.target>17</maven.compiler.target>
    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
  </properties>

</project>
```

**Comment by rnaval** [comment]
Hello @suztomo! Tried out your suggestion for the proto- libraries, but still getting an exception.
Here's the original dependency for the proto- library being used:
```
  <dependencyManagement>
    <dependencies>
      <dependency>
        <groupId>com.google.cloud</groupId>
        <artifactId>libraries-bom</artifactId>
        <version>26.17.0</version>
        <type>pom</type>
        <scope>import</scope>
      </dependency>
    </dependencies>
  </dependencyManagement>

  <dependencies>
    <dependency>
      <groupId>junit</groupId>
      <artifactId>junit</artifactId>
      <scope>test</scope>
    </dependency>
    <dependency>
      <groupId>org.mockito</groupId>
      <artifactId>mockito-core</artifactId>
      <scope>test</scope>
    </dependency>
    <dependency>
      <groupId>com.google.cloud</groupId>
      <artifactId>google-cloud-monitoring</artifactId>
    </dependency>
    <dependency>
      <groupId>com.google.cloud</groupId>
      <artifactId>google-cloud-compute</artifactId>
    </dependency>
    <dependency>
      <groupId>com.google.cloud</groupId>
      <artifactId>google-cloud-storage</artifactId>
    </dependency>
    <dependency>
      <groupId>com.google.cloud</groupId>
      <artifactId>google-cloud-filestore</artifactId>
    </dependency>
    <dependency>
      <groupId>com.google.protobuf</groupId>
      <artifactId>protobuf-java-util</artifactId>
      <version>4.0.0-rc-2</version>
    </dependency>
  </dependencies>
```

I removed the version, and looked at the dependency tree being generated - seems to be updated:
```
[INFO] +- com.google.protobuf:protobuf-java-util:jar:3.23.2:compile
```

But still getting an exception somewhere in the client:
```
java.lang.IllegalAccessError: class com.google.cloud.compute.v1.VpnGatewayAggregatedList tried to access method 'com.google.protobuf.LazyStringArrayList com.google.protobuf.LazyStringArrayList.emptyList()' (com.google.cloud.compute.v1.VpnGatewayAggregatedList is in unnamed module of loader java.net.URLClassLoader @18078bef; com.google.protobuf.LazyStringArrayList is in unnamed module of loader 'app')
	at com.google.cloud.compute.v1.VpnGatewayAggregatedList.<init>(VpnGatewayAggregatedList.java:466)
	at com.google.cloud.compute.v1.VpnGatewayAggregatedList.<clinit>(VpnGatewayAggregatedList.java:2186)
	at com.google.cloud.compute.v1.stub.HttpJsonVpnGatewaysStub.<clinit>(HttpJsonVpnGatewaysStub.java:126)
	at com.google.cloud.compute.v1.stub.VpnGatewaysStubSettings.createStub(VpnGatewaysStubSettings.java:334)
	at com.google.cloud.compute.v1.VpnGatewaysClient.<init>(VpnGatewaysClient.java:149)
	at com.google.cloud.compute.v1.VpnGatewaysClient.create(VpnGatewaysClient.java:131)
	at com.google.cloud.compute.v1.VpnGatewaysClient.create(VpnGatewaysClient.java:123)
```

I'll try and create a minimum project to reproduce this and share with you.

**Comment by suztomo** [comment]
> ```
>      <groupId>com.google.protobuf</groupId>
>      <artifactId>protobuf-java-util</artifactId>
>      <version>4.0.0-rc-2</version>
> ```

protobuf-java-util is also declared in the libraries-bom. Our SDK is not built or tested with that `4.0.0-rc-2` version. Would you remove `<version>4.0.0-rc-2</version>`?

**Comment by rnaval** [comment]
Hello @suztomo! Yeah I removed that and the new dependency was 3.23.2 as mentioned in the earlier comment:

<snip>
I removed the version, and looked at the dependency tree being generated - seems to be updated:
```
[INFO] +- com.google.protobuf:protobuf-java-util:jar:3.23.2:compile
```
<snip>

But still throwing an exception.

**Comment by suztomo** [comment]
OK. Then I'll need a minimum project to reproduce the error.

> I'll try and create a minimum project to reproduce this and share with you.

Thank you.

**Comment by rnaval** [comment]
Hello @suztomo - I'm trying to create a small project to reproduce this issue, but I can't seem to get it to occur in a smaller scale.

One thing I noticed though is when I tried to reproduce it via a unit test using the original code, the issue cannot be reproduced.

I'm not sure if there are any more library conflicts - I'm still looking through the dependency graphs for this.

I'll try and get to the bottom of this and give an update soon. Thanks.

**Comment by suztomo** [comment]
> One thing I noticed though is when I tried to reproduce it via a unit test using the original code, the issue cannot be reproduced.

Nice observation. Would you try to focus the class path (list of JAR files) of the problematic case? It's possible that the problematic case runs in a different set of JAR files than your Maven project's JAR files.

**Comment by suztomo** [comment]
Do you use Dataproc or Hadoop environment?

**Comment by rnaval** [comment]
@suztomo This seems to have been resolved by an update similar to #9625 - I can now use 26.19.0 without any issues. Thanks!

**Comment by suztomo** [comment]
Thank you for update.
